### DISCUSSION TITLE
Your Help Is Needed: Please Share Your Data Object Layouts for Our New UI! 🚀

### DISCUSSION THREAD

**Discussion by idaiv** [pimcore/pimcore#16349] [General]
We're diving into the UI redesign, and could use some of your help. We're keen to see how you configure data objects editor layouts! 

Please share a screenshot showcasing your setup with diverse elements like tabs, localized fieds, blocks, field collections and more. It is primary about the overall layout, not about any particular data fields etc. So of course, you can blur sensitive parts of the layouts (texts, labels, data values, ...). 
 
Additionally, you can share:
What challenges you are facing with the data objects layouts?
Do you have specific expectations or wishes for the new UI in regard to the data objects layouts?
If there is any feedback you have from users.
 
Feel free to attach screenshots or simply describe your setup in a reply to this thread.
 
Thanks a ton for contributing to our UI journey!

Reference:
![image (5)](https://github.com/pimcore/pimcore/assets/138690998/72e64601-e67c-4b4e-8a4f-9caf31196480)

**Comment by NiklasBr** [comment]
Accessibility, accessibility, accessibility! It's actual law both across Europe and USA, both for private and public sectors in most places as well. It's about time to get the admin interface up to proper WCAG standards.

**Comment by kneumann37** [comment]
These are some of our examples. We often use tabs and addable elements. The shown product group data object is used to build a content page.

<img width="1129" alt="productgroup1" src="https://github.com/pimcore/pimcore/assets/149158162/e9129e16-aabf-4eb6-9ccf-c58f105fdb3d">
<img width="1130" alt="productgroup2" src="https://github.com/pimcore/pimcore/assets/149158162/7ac0d260-baf6-41da-ba20-407ef023162e">

We and the editors always have a little UX trouble when a data objects uses single language as well as multi language fields, because it is hard to recognize by the indentation. Also to avoid having the line of language tabs over and over again you need to group multi language fields which sometimes takes them out of their context.

<img width="1130" alt="translation" src="https://github.com/pimcore/pimcore/assets/149158162/32fa1e1c-eb10-4647-91e4-476d0d9d38c3">

**Reply by idaiv** [reply]
Thank you for the input @kneumann37! Yes, the multi language vs single language fields are a topic we are currently working on.

**Comment by bluxi7** [comment]
I generally find it hard to align fields and to use all the horizontal screen space available. Sometimes it feels like the CSS field in the class editor is not working (or at least it's not very obvious why changes don't have an effect).

Also, in the class editor, it's confusing to me why certain elements can't be nested via the context menu but it works by drag and drop.

**Comment by alexej-d** [comment]
- Setting relative widths or adjusting elements to fill the horizontal space is very error-prone and sometimes impossible.
- The region element is named strangely: I always expect it to be a region I would nest in a panel, whereas technically it is the other way around.
- Lots of elements need to be adjusted manually to position their left or right boundaries inline.
- Multiselect lists have no indicator when they are scrollable.
- Sometimes I want to globally adjust the layout of specific data elements, but for these cases I would need to deeply dive into ExtJS. It would be great if each data element had a twig template that one could extend to make simple changes.

I know this does not belong in here, but I would also love a brick functionality like the one in documents and snippets. Right now, I either create relations to snippets or use field collections for cases like that, but some kind of areablock element that uses all existing bricks would be great.

**Comment by beheist** [comment]
Here is an example from one of our current projects.

![image](https://github.com/pimcore/pimcore/assets/10347669/08af2d80-2aaf-49d5-aaff-caa132ff9e68)

My biggest issue with the UI, apart from what was already mentioned related to the multilanguage fields, is inheritance. Once you overwrite a field, it is not possible to "reset" the value to make the object inherit from the parent again, at least not for all data types.
